Dependencies/Axiom/Verify-AssertionFailed.ps1

```diff
@@ -4,24 +4,37 @@ function Verify-AssertionFailed {
         [ScriptBlock]$ScriptBlock
     )
 
-    $assertionExceptionThrown = $false
     $err = $null
+    $assertionExceptionThrown = $false
     try {
         $null = & $ScriptBlock
     }
     catch [Exception] {
         $assertionExceptionThrown = ($_.FullyQualifiedErrorId -eq 'PesterAssertionFailed')
         $err = $_
-        $err
+        Write-Output $err
```

Comment:
What is the reason to use `Write-Output` here?

Reply:
There is no change of behavior there, it was just a matter of making more clear the intent that the `$err` variable is to be sent as output to the pipeline.

I can revert that, but I think that is something that we should discuss as part of our coding practices. It could be in a separate issue of course.